### DISCUSSION TITLE
How to hide other items in a similar way as hiding a plotted line by clicking the legend at the corner?

### DISCUSSION THREAD

**Discussion by PythonAI1234** [highfestiva/finplot#499] [Q&A] [answered]
I modified the programme and drew some arrows by using a method similar to how the program draws candles with ohlc.
I put some numbers into a dataframe with the same length as the candles, it's like a buffer.
I only put value in some index while the rest remains empty.
Now I want to hide them just like I hide a line I draw with fplt.plot(ma30, ax=ax, legend='ma30'), by clicking on the legend on the corner.
I couldn't figure out how this works for the line.

If you could provide an example on how to change the programme to hide the candle sticks like I hide lines, I would know what to do for my special case.

Thank you

**Comment by highfestiva** [comment, marked as answer]
Try this:
```python
candles = fplt.candlestick_ochl(..., ax=ax)

def key_pressed(vb, ev):
    if ev.text() == '1':
        ax.removeItem(candles)
    if ev.text() == '2':
        ax.addItem(candles)
    old_key_pressed(vb, ev)

old_key_pressed, fplt._key_pressed = fplt._key_pressed, key_pressed
```

**Reply by PythonAI1234** [reply]
That was helpful, thank you.
